tools/depends/configure.ac

```diff
@@ -703,6 +725,10 @@ else
   platform_cxxflags="$platform_cxxflags_release"
 fi
 
+if test "$ffmpeg_options" == "default"; then
```

Comment:
maybe should be `"x$ffmpeg_options" == "xdefault"`

Reply:
not needed. It's fine as is.